modules/udf/README.md

```diff
@@ -3,8 +3,12 @@
 This submodule adds some utility [user defined functions](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions)
 you may find useful when building BigQuery queries.
 
-To add these to your dataset, simply enable them in the BigQuery module:
+Example:
 
-```
-  add_udf_utilities = true
-```
+module "add_udfs" {
+  source = source  = "terraform-google-modules/bigquery/google//modules/udf"
+  version = "~> 3.0"
+
+  dataset_id = "example_dataset"
```

Comment:
Please show using this with the main module. In particular, we should make sure these are both *references* to the main module as that dependency order is important.